finance-api/user-guides/fetch-payout-info.md

```diff
@@ -0,0 +1,54 @@
+Fetch payout information
+===
+Using the Finance API, you can fetch payout information from a merchant's Zettle account. By default, it's fetched from the merchant's Zettle liquid account. The payout information includes the account balance, currency, payout, periodicity, and so on. 
```

Comment:
> By default, it's fetched from the merchant's Zettle liquid account.

We don't do payouts from preliminary account. We don't need this sentence.

Reply:
Better to say "payout amount" when listing all the attributes.
 Is "so on" - `discountRemaining`?  If yes why not just put it there?